examples/test_50d.py

```diff
@@ -12,29 +12,27 @@ def __init__(self,dim=50):
         self.dim=dim
         self.names=['{0}'.format(i) for i in range(self.dim)]
         self.bounds=[[-10,10] for _ in range(self.dim)]
-#        self.bounds[0] = [-50, 50]
         self.analytic_log_Z=0.0 - sum([np.log(self.bounds[i][1]-self.bounds[i][0]) for i in range(self.dim)])
 
     def log_likelihood(self,p):
-        return np.sum([-0.5*p[n]**2-0.5*np.log(2.0*np.pi) for n in p.names])##np.sum([self.distr.logpdf(p[n]
```

Comment:
Why did you change the way of getting the parameter? Isn't this going to be slower?